src/services/http.service.ts

```diff
@@ -10,15 +10,17 @@ export type HttpRequestConfig = AxiosRequestConfig;
 export default class Http {
   static async get<T>(
     url: string,
-    config: HttpRequestConfig = {}
+    config: HttpRequestConfig = {},
+    headers: Record<string,string> = requestHeaders
```

Comment:
You don't need to add a parameter for header, you can include them in the request config